lib/teaspoon/configuration.rb

```diff
@@ -152,6 +153,12 @@ def self.formatters
       @@formatters.to_s.split(/,\s?/)
     end
 
+    def self.coverage_ignored_files
+      return [] if @@coverage_ignored_files.blank?
```

Comment:
This was a case of copying the way which `formatters` work. You're right, if it were blank it would be an empty array anyway.